.circleci/config.yml

```diff
@@ -244,6 +244,12 @@ jobs:
       - image: cimg/ruby:3.1.2
     steps:
       - checkout
+      - revenuecat/install-gem-mac-dependencies:
```

Comment:
Hmm isn't this a unix machine? Should we be using `revenuecat/install-gem-unix-dependencies`.

Reply:
Ohh right! My mistake. Thank you! Changed in https://github.com/RevenueCat/purchases-capacitor/pull/578/commits/8130f1a5e236ba867d56ea113cc289fbc3f1ed85